### ISSUE TITLE
Source data/input data request for Order Reference: PAC-19-29262515-1764776 Keras Deep Learning Cookbook [eBook]  

### ISSUE THREAD

**Issue by DeeDaiDaniels** [ml-resources/deeplearning-keras#7]
Keras Deep Learning Cookbook [eBook]  
Dear Sir/Madam,
I would like to have access to download all of the source data to windows env (not linux/unix env) for this book I purchased a couple of days ago.Thanks. Yueying
  The data for this recipe can be found at https://www.kaggle.com/c/ intel-mobileodt-cervical-cancer-screening. The dataset is part of the challenge to develop an algorithm that accurately identifies a woman's cervix type based on images. Doing so will prevent ineffectual treatments and allow healthcare providers to give a proper referral for cases that require more advanced treatment. As a first step, clone the GitHub repository at https://github.com/ml-resources/ deeplearning-keras/tree/ed1/ch05. Download and save the training and test sets to the data folder in the repository: train.7z: The training set. The images are organized in their labeled categories: Type_1, Type_2, and Type_3. test.7z: The test set.
